go/deps.bzl

```diff
@@ -0,0 +1,4 @@
+load("@bazel_gazelle//:deps.bzl", "go_repository")
+
+def go_dependencies():
```

Comment:
What is it for?

Reply:
This is for any files we import with go get and then generate with `gazelle-update-dependencies` ( or those we want to specify manually).

In the recent commit I have included those generated for `godogs`.